test/kubemark/start-kubemark.sh

```diff
@@ -126,17 +147,25 @@ gcloud compute ssh --zone="${ZONE}" --project="${PROJECT}" "${MASTER_NAME}" \
     sudo bash -c \"echo \"${KUBE_PROXY_TOKEN},kube_proxy,kube_proxy\" >> /srv/kubernetes/known_tokens.csv\" && \
     sudo bash -c \"echo ${password},admin,admin > /srv/kubernetes/basic_auth.csv\""
 
+writeEnvironmentFiles
+
 if [ "${RUN_FROM_DISTRO}" == "false" ]; then
   gcloud compute copy-files --zone="${ZONE}" --project="${PROJECT}" \
     "${KUBE_ROOT}/_output/release-tars/kubernetes-server-linux-amd64.tar.gz" \
     "${KUBEMARK_DIRECTORY}/start-kubemark-master.sh" \
     "${KUBEMARK_DIRECTORY}/configure-kubectl.sh" \
+    "${RESOURCE_DIRECTORY}/api_server_flags" \
+    "${RESOURCE_DIRECTORY}/scheduler_flags" \
+    "${RESOURCE_DIRECTORY}/controllers_flags" \
     "${MASTER_NAME}":~
 else
   gcloud compute copy-files --zone="${ZONE}" --project="${PROJECT}" \
     "${KUBE_ROOT}/server/kubernetes-server-linux-amd64.tar.gz" \
     "${KUBEMARK_DIRECTORY}/start-kubemark-master.sh" \
     "${KUBEMARK_DIRECTORY}/configure-kubectl.sh" \
+    "${RESOURCE_DIRECTORY}/api_server_flags" \
+    "${RESOURCE_DIRECTORY}/scheduler_flags" \
+    "${RESOURCE_DIRECTORY}/controllers_flags" \
```

Comment:
Are you planning to also pipe args to hollow-node? This would be super useful for protobufs.